lepidopter-fh/etc/ooniprobe.conf

```diff
@@ -62,6 +72,10 @@ tor:
     # This should be set to something to avoid having Tor download each time
     # the descriptors and consensus data.
     data_dir: /opt/ooni/tor_data_dir
+    #
+    # This is the timeout after which we consider to to not have
+    # bootstrapped properly.
+    timeout: 200
 torrc:
     #HTTPProxy: host:port
     #HTTPProxyAuthenticator: user:password
```

Comment:
As I noted in another comment this file should only include lepidopter specific configuration variables.

It should therefore look something like:

```
basic:
   logfile: /var/log/ooni/ooniprobe.log
advanced:
   webui_port: 8842
   webui_address: "0.0.0.0"
tor:
    data_dir: /opt/ooni/tor_data_dir
    timeout: 200
```